scene/theme/theme_db.cpp

```diff
@@ -370,6 +371,19 @@ void ThemeDB::update_class_instance_items(Node *p_instance) {
 	}
 }
 
+void ThemeDB::get_class_own_items(const StringName &p_class_name, List<ThemeItemBind> *r_list) {
+	HashMap<StringName, HashMap<StringName, ThemeItemBind>>::Iterator E = theme_item_binds.find(p_class_name);
+	if (E) {
+		for (const KeyValue<StringName, ThemeItemBind> &F : E->value) {
+			if (F.value.external || F.value.class_name != p_class_name) {
```

Comment:
When the class name can be different if the item is not external?

Reply:
For the moment it can't, but this permutation is possible so I thought I'd add a check.

Reply:
I actually need to rework this method, and I will need to use this check, because I'm going to sort through inherited classes.